pkg/volume/csi/csi_plugin.go

```diff
@@ -888,3 +990,15 @@ func isV1Version(version string) bool {
 
 	return parsedVersion.Major() == 1
 }
+
+// ToPluginHandler converts the generic volume plugin to a specific csiPlugin and
+// returns that as a PluginHandler. In case the generic plugin is not a CSI
```

Comment:
`In case the generic plugin is not a CSI...` will this ever be the case ?

Reply:
Please see [the other comment](https://github.com/kubernetes/kubernetes/pull/73919#discussion_r269059833).